src/DynamoCore/Graph/Nodes/NodeCategories.cs

```diff
@@ -420,8 +421,8 @@ internal static string MakeRelativePath(string basePath, string subjectPath)
             var documentUri = new Uri(basePath, UriKind.Absolute);
             var assemblyUri = new Uri(subjectPath, UriKind.Absolute);
 
-            var relativeUri = documentUri.MakeRelativeUri(assemblyUri);
-            var relativePath = relativeUri.OriginalString.Replace('/', '\\');
+            var relativeUri = documentUri.MakeRelativeUri(assemblyUri).OriginalString;
+            var relativePath = WebUtility.UrlDecode(relativeUri.Replace("+", "%2B")).Replace('/', '\\');
```

Comment:
@gregmarr can you also explain the logic here ... not exactly sure what is going on here given that your description states url decoding is removed from display?

* Should a comment get added to this method summary about this change?

* Have you tried running the tests locally?